docs/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+**Implementation details:**
```

Comment:
This should go into `user-visible changes` for the next release below, not here at the beginning of the file.
Add some more explanation about the feature and who should use it.

Reply:
Okay, I see the future release now! Previously, I don't know the release date and how to name the release.